### DISCUSSION TITLE
Run shell command and get output texts

### DISCUSSION THREAD

**Discussion by JmyL** [quarto-dev/quarto-cli#7172] [Q&A]
### Description

I would like to run shell command and get it‘s output in console like python. This could be little bit dangerous but pretty useful for me.

**Comment by mcanouil** [comment]
See:
- https://github.com/quarto-dev/quarto-cli/discussions/5275
- https://github.com/quarto-dev/quarto-cli/discussions/4634

Not more dangerous than running Python which has access to bash/shell.
